galaxy_api/api/models.py

```diff
@@ -0,0 +1,96 @@
+from django.db import models
+
+from galaxy_api import constants as const
+from galaxy_api.auth import models as auth_models
+
+
+__all__ = (
+    'Namespace',
+    'NamespaceLink',
+    'Collection',
+)
+
+
+class Namespace(models.Model):
+    """
+    A model representing Ansible content namespace.
+
+    Fields:
+        name: Namespace name. Must be lower case containing only alphanumeric
+            characters and underscores.
+        company: Optional namespace owner company name.
+        email: Optional namespace contact email.
+        logo_url: Optional namespace logo URL.
+        description: Namespace brief description.
+        resources_page_src: Namespace resources page in markdown format.
+        resource_page_html: HTML representation of a resources page.
+
+    Relations:
+        owners: Reference to namespace owners.
+        links: Reference to related links.
+
+    """
+
+    # Fields
+
+    name = models.CharField(max_length=const.MAX_NAME_LENGTH, unique=True, editable=False)
+    company = models.CharField(max_length=64, blank=True)
+    email = models.CharField(max_length=256, blank=True)
+    logo_url = models.CharField(max_length=256, blank=True)
```

Comment:
Lets use `avatar_url` here, to try and maintain better compatibility with galaxy.

Reply:
Ok